addons/payment_authorize/controllers/main.py

```diff
@@ -17,8 +17,17 @@ class AuthorizeController(http.Controller):
     @http.route([
         '/payment/authorize/return/',
         '/payment/authorize/cancel/',
-    ], type='http', auth='public', csrf=False)
+    ], type='http', auth='public', csrf=False, save_session=False)
     def authorize_form_feedback(self, **post):
+        """ Process the data returned by Authorize after redirection.
+                The route is flagged with `save_session=False` to prevent Odoo from assigning a new session
+                to the user if they are redirected to this route with a POST request. Indeed, as the session
+                cookie is created without a `SameSite` attribute, some browsers that don't implement the
+                recommended default `SameSite=Lax` behavior will not include the cookie in the redirection
+                request from the payment provider to Odoo. As the redirection to the '/payment/status' page
+                will satisfy any specification of the `SameSite` attribute, the session of the user will be
+                retrieved and with it the transaction which will be immediately post-processed.
+                """
```

Comment:
```suggestion
        """ Process the data returned by Authorize after redirection.
        The route is flagged with `save_session=False` to prevent Odoo from assigning a new session
        to the user if they are redirected to this route with a POST request. Indeed, as the session
        cookie is created without a `SameSite` attribute, some browsers that don't implement the
        recommended default `SameSite=Lax` behavior will not include the cookie in the redirection
        request from the payment provider to Odoo. As the redirection to the '/payment/status' page
        will satisfy any specification of the `SameSite` attribute, the session of the user will be
        retrieved and with it the transaction which will be immediately post-processed.
        """
```

Reply:
Mind the blank line after the docstring title and the indentation =]